packages/types/src/customer/mutations.ts

```diff
@@ -283,26 +283,6 @@ export interface CustomerUpdatableFields {
   metadata?: Record<string, unknown> | null
 }
 
-/**
- * The customer group to be created.
- */
-export interface CreateCustomerGroupDTO {
-  /**
-   * The name of the customer group.
-   */
-  name: string
-
-  /**
-   * Holds custom data in key-value pairs.
-   */
-  metadata?: Record<string, unknown> | null
-
-  /**
-   * Who the customer group.
-   */
-  created_by?: string
-}
-
```

Comment:
clarification: this interface is defined twice in the same file. So, I removed one of the definitions